src/components/Tile.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, getByTestId } from '@testing-library/react';
+import Tile from './Tile';
+
+test('renders button with exclamation icon', () => {
+    const { container } = render(<Tile icon='exclamation' tileWidth="100" finder="" />);
+    const buttonElement = getByTestId(container, "exclamation");
+    expect(buttonElement).toBeInTheDocument();
+});
+
+test('renders button with bomb icon', () => {
+    const { container } = render(<Tile icon='bomb' tileWidth="100" finder="" />);
+    const buttonElement = getByTestId(container, "bomb");
+    expect(buttonElement).toBeInTheDocument();
+});
+
+test('renders button with radiation icon', () => {
+    const { container } = render(<Tile icon='radiation' tileWidth="100" finder="p1" />);
```

Comment:
Can we add default value of tileWidth=100 at component level directly???

Reply:
No the tilewidth is sent from the parent component but for test simulation purpose as we are rendering only one tile I defaulted the value to be 100